native/src/main/java/tech/sourced/babelfish/JsonASTVisitor.java

```diff
@@ -36,9 +37,12 @@
     private boolean verboseJson = false;
     private HashSet<String> skipMethods;
     private Hashtable<String, Vector<ChildrenTypeCacheValue>> childrenMethodsCache;
-    private Vector<String> jsonDebugLog;
-    private IASTNode lastTypeVisited = null;
+    private Stack<IASTDeclSpecifier> typesVisited;
```

Comment:
Please document at least the new entry here. I assume this is meant to be the stack of types on the path from the root to the current point in the traversal, but I'm not sure.

Reply:
Done in next commit.